mailbox/store/src/main/java/org/apache/james/mailbox/store/event/MailboxEventDispatcher.java

```diff
@@ -163,8 +165,13 @@ public void aclUpdated(MailboxSession session, MailboxPath mailboxPath, ACLDiff
         listener.event(eventFactory.aclUpdated(session, mailboxPath, aclDiff, mailboxId));
     }
 
-    public void moved(MailboxSession session, MessageMoves messageMoves, Map<MessageUid, MailboxMessage> messages) {
-        MessageMoveEvent moveEvent = eventFactory.moved(session, messageMoves, messages);
+    public void moved(MailboxSession session, MessageMoves messageMoves, Collection<MessageId> messageId) {
+        MessageMoveEvent moveEvent = MessageMoveEvent.builder()
```

Comment:
it's not consistent with other event creation. Why did you decided to remove the method from EventFactory ?

Reply:
Because it was not adding value compared to building the event by hand. I can re-add this if you want.

Reply:
I see at least two values:
* being consistent
* have a single point to look for building all events

BTW, we could very well only provide builders from EventFactory so have the best of both world (but I suspect it's not for today)

Reply:
I re-added the method

> (but I suspect it's not for today)

Yes, sorry about that, but let's keep track of this idea... That 'might' be valuable as a preliminary step  to not use MailboxEventDispatcher anymore but rely on EventBus